dataland-frontend/tests/e2e/utils/SfdrUpload.ts

```diff
@@ -59,3 +61,11 @@ export function uploadCompanyAndSfdrDataViaApi(
     },
   );
 }
+/**
+ * Adds reports to the dataset via the Sfdr upload form for the given dataset
+ */
+export function addsReportsToSfdrDataViaForm(): void {
```

Comment:
I'd prefer something like "selectsReportsForUploadInSfdrForm" because it is only a selection, not an upload.

Reply:
renamed

Reply:
👍